specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,109 @@
+- Feature Name: Email consent opt-in and Unsubscribe page
+- Created: 2024-02-21
+- Associated Issue: #3109
+
+# Summary
+
+Work required to enable email communication with Pontoon users: opt-in consent user settings, methods for collecting user opt-in, unsubscribe method via a single page that doesn't require login.
+
+# Motivation
+
+There does not exist currently an email communication consent opt-in for users who register with Pontoon. This restricts our ability to reach contributors outside of the platform and reduces our effectiveness in keeping localizers informed and engaged. So we need to create settings that allow users to control their ability to opt in to email messages.
+
+This will allow us to reach out within legal compliance to the wider audience of registered users for things such as surveys, online events, etc. Future major platform enhancements will be able to build off this work, and starting this early will provide enough time to ensure interested and active contributors are opted-in ahead of time.
+
+Value for users / problems solved:
+- Users can select their preference for email communication
+- Engagement/reach expanded beyond logged-in active Pontoon users
+
+
+# Feature explanation
+
+## Email communication user settings
```

Comment:
Are we going to include mockups in the spec?

Reply:
I created *very* rough mock ups for the opt-in consents for new and existing users.

Reply:
Thanks, these are very helpful! I think we should streamline the copy on these pages to make the CTA stand out more.

I'm happy to take a stab at that and work on the missing mockups (we should have one at least for the Unsubscribe page).